<?xml version="1.0"?>
<launch>

  <param name="debug"          value="True"               />
  <param name="apps_root"      value="$(find agile_apps)" />
  <param name="libs_root"      value="/home/bibei/Workspaces/agile_ws/devel/lib" />
  <param name="pkgs_root"      value="/home/bibei/Workspaces/agile_ws/src/agile_robot" />
  <rosparam command="load" file="$(find agile_apps)/config/leg_agile_robot.yaml" />
 
  <!-- Load hardware interface -->
  <node name="agile_robot" pkg="agile_apps" type="agile_apps_robot" output="screen" />

</launch>

